packages/manager/modules/emailpro/src/domain/dkim-autoconfig/emailpro-domain-dkim-autoconfig.controller.js

```diff
@@ -144,4 +187,14 @@ export default class EmailProDomainDkimAutoconfigCtrl extends DkimAutoConfigurat
         this.services.$scope.resetAction();
       });
   }
+
+  getBodyText(status) {
+    let translationKey = STATUS_TO_TEXT_MAP[status] || '';
+    if (status === this.GLOBAL_DKIM_STATUS.NOT_CONFIGURED) {
+      translationKey = this.domainDiag.isOvhDomain
+        ? 'emailpro_tab_domain_diagnostic_dkim_activation_ovhcloud'
+        : 'emailpro_tab_domain_diagnostic_dkim_activation_no_ovhcloud';
+    }
+    return this.services.$translate.instant(translationKey);
```

Comment:
```suggestion
    let translationKey;
    if (status === this.GLOBAL_DKIM_STATUS.NOT_CONFIGURED) {
      translationKey = this.domainDiag.isOvhDomain
        ? 'emailpro_tab_domain_diagnostic_dkim_activation_ovhcloud'
        : 'emailpro_tab_domain_diagnostic_dkim_activation_no_ovhcloud';
    } else {
        translationKey = STATUS_TO_TEXT_MAP[status];
     }
    return translationKey ? this.services.$translate.instant(translationKey) : '';
```
To avoid useless initialization

Reply:
I've done it, 
but can you explain to me why it was necessary to do this?